react/week4/github_users_search/src/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import UserList from "./UserList";
+//import App from './App';
+import searchContext from "./context";
+import * as serviceWorker from './serviceWorker';
+
+const root = document.getElementById('root');
+const ContextProvider = searchContext.Provider;
+
+function fetchGithubUsers(query) {
+  if (query !== '') {
+    const URL = `https://api.github.com/search/users?q=${query}`
+    return fetch(URL).then((response) => {
+      if (response.status >= 200 && response.status <= 299) {
```

Comment:
I understand the intent here, and it is great that you looked for which status are considered ok.

With that said, if I was reviewing this, I would wonder: "why would this code need to accept 204"?

It is recommended to use `response.status === 200` here, since any other 2XX response is likely not what we are trying to get from this simple GET.

It also makes it clear for readers that we are really just expecting a 200.

Reply:
Yes it makes sense. I will remove them :)